Templates/colors.json

```diff
@@ -0,0 +1,4687 @@
+        { "name": "AriseTime", "type": "float32" },
+        { "name": "Phase", "type": "float32" },
+        { "name": "FloorSize", "type": "ObjLightTimeFloorSpawner::FloorSize" },
+        { "name": "ActionType", "type": "ObjLightTimeFloorSpawner::ActionType" },
+        { "name": "EventType", "type": "ObjLightTimeFloorSpawner::EventType" }
+      ]
+    },
+    "ObjLongTankSpawner": {
+      "fields": [
+      ]
+    },
+    "ObjMeteorSpawner": {
+      "fields": [
+        { "name": "SizeType", "type": "ObjMeteorSpawner::SizeType" }
+      ]
+    },
+    "ObjPotSpawner": {
+      "fields": [
+        { "name": "UpSpeed", "type": "float32" },
+        { "name": "SpeedMin", "type": "float32" },
+        { "name": "SpeedMax", "type": "float32" }
+      ]
+    },
+    "ObjPushBlockSpawner": {
+      "fields": [
+        { "name": "AppearTime", "type": "float32" },
+        { "name": "DisappearTime", "type": "float32" },
+        { "name": "AppearStopTime", "type": "float32" },
+        { "name": "DisappearStopTime", "type": "float32" },
+        { "name": "Phase", "type": "float32" },
+        { "name": "WaitTime", "type": "float32" },
+        { "name": "AppearDistance", "type": "float32" },
+        { "name": "BlockType", "type": "ObjPushBlockSpawner::BlockType" },
+        { "name": "IsEventStart", "type": "bool" },
+        { "name": "IsPushDead", "type": "bool" }
+      ]
+    },
+    "ObjRollerCoasterSpawner": {
+      "fields": [
+        { "name": "PathID", "type": "uint32" },
+        { "name": "StartPosition", "type": "float32" },
+        { "name": "MaxSpped", "type": "float32" },
+        { "name": "FristSpeed", "type": "float32" },
+        { "name": "Accel", "type": "float32" },
+        { "name": "VehicleNum", "type": "uint32" },
+        { "name": "Type", "type": "ObjRollerCoasterSpawner::Type" },
+        { "name": "PrepareSpeed", "type": "float32" }
+      ]
+    },
+    "ObjRollerCoasterCollisionSpawner": {
+      "fields": [
+        { "name": "CollisionWidth", "type": "float32" },
+        { "name": "CollisionHeight", "type": "float32" }
+      ]
+    },
+    "ObjChaserBrokenBlockSpawner": {
+      "fields": [
+      ]
+    },
+    "ObjColorFilterSpawner": {
+      "fields": [
+        { "name": "CollisionWidth", "type": "float32" },
+        { "name": "CollisionHeight", "type": "float32" },
+        { "name": "FadeSpeed", "type": "float32" },
+        { "name": "FadeTime", "type": "float32" },
+        { "name": "TopFadeSpeed", "type": "float32" }
+      ]
+    },
+    "ObjDenergySpawner": {
+      "fields": [
+        { "name": "FirstSpeed", "type": "float32" },
+        { "name": "SecondSpeed", "type": "float32" }
+      ]
+    },
+    "ObjFloatingFloorSpawner": {
+      "fields": [
+        { "name": "Type", "type": "ObjFloatingFloorSpawner::Type" },
+        { "name": "MinSpeed", "type": "float32" },
+        { "name": "MaxSpeed", "type": "float32" },
+        { "name": "HideEffect", "type": "bool" }
+      ]
+    },
```

Comment:
Parameters named based on parameters with similar functionality on other objects and several in-game funcitons:
(0x802BADF4, NTSC-J Wii)
(0x802BA9A4, NTSC-J Wii)
(0x802BAF00, NTSC-J Wii)